.travis.yml

```diff
@@ -9,7 +9,8 @@ cache:
 php:
     - 7.1
     - 7.2
-    - 7.3
```

Comment:
💡 I've removed the 7.3 run here because `matrix.include` already contains a run on PHP 7.3, so this one was redundant.